analyzer-comments/elixir/newsletter/send_newsletter_does_not_call_write.md

```diff
@@ -0,0 +1,3 @@
+# send newsletter does not call write
+
+Make sure that `send_newsletter/3` does not call `File.write/2` directly, but rather uses `log_sent_email/2` to write to the log file.
```

Comment:
Is this check necessary if "send_newsletter reuses functions" will also ensure the usage of `log_sent_email` in `send_newsletter`?

Reply:
I guess it's not really necessary anymore... 
Although I think I set this one as `:necessary` but the others "X uses Y" as `:actionable`. It's kind of arbitrary... But yeah, I will remove this one!